Story/CCP-225

mkarthik90 opened the pull request.

diging-jenkins commented:
Can one of the admins verify this patch?

mkarthik90 commented:
Need to include the changes for creating wrappers from concept list screen.

diging-jenkins commented:
Can one of the admins verify this patch?

jdamerow commented on Conceptpower+Spring/src/main/webapp/WEB-INF/views/home.jsp:
```diff
@@ -141,10 +142,29 @@ function prepareMergeConcept(conceptId) {
+      success : function(details) {
+        $.each(details.conceptEntries, function (index, conceptEntry) {
+          window.location = '${pageContext.servletContext.contextPath}/auth/conceptlist/addconceptwrapper?word=' + conceptEntry.lemma + '&selectedConceptList=' + conceptEntry.conceptList + '&description=' + conceptEntry.description + '&selectedType=' + conceptEntry.type + '&wrapperids=' + conceptEntry.id;
+        });
```
I'm pretty sure you have all this info when you build the results page, why do you make an extra concept request?

mkarthik90 commented:
@jdamerow  Addressed review comments. Removed ajax call and replaced with the values from concept entry. Passed the individual values instead of concept entry object to the method. For some reason js is not able to process the concept entry object so passed the individual values to createWrapper() method.

diging-jenkins commented:
Can one of the admins verify this patch?

jdamerow commented:
ok to test
